From phil@ipom.com Sun Oct 23 23:57:05 2005
Message-ID: <435C84B7.2010303@ipom.com>
Date: Sun, 23 Oct 2005 23:52:39 -0700
From: Phil Dibowitz <phil@ipom.com>
To: Greg KH <greg@kroah.com>
CC: Stuart Black <stuart_black@yahoo.co.uk>
Subject: USB Storage: Expand range of Freecom unusual_devs entry


Freecom seems to be one of those vendors that can't get the GET CAPACITY
thing right. This expands the US_FL_FIX_CAPACITY flag for the entire
range of their fccd product line.

This is based on a patch sent by Stuart Black
<stuart_black@yahoo.co.uk>.

Signed-off-by: Phil Dibowitz <phil@ipom.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/unusual_devs.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/storage/unusual_devs.h
+++ gregkh-2.6/drivers/usb/storage/unusual_devs.h
@@ -763,7 +763,7 @@ UNUSUAL_DEV(  0x07ab, 0xfc01, 0x0000, 0x
 #endif
 
 /* Reported by Eero Volotinen <eero@ping-viini.org> */
-UNUSUAL_DEV(  0x07ab, 0xfccd, 0x0406, 0x0406,
+UNUSUAL_DEV(  0x07ab, 0xfccd, 0x0000, 0x9999,
 		"Freecom Technologies",
 		"FHD-Classic",
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
